### ISSUE TITLE
How to get access to screenshots created inside Docker?

### ISSUE THREAD

**Issue by MRizkBV** [jef/streetmerchant#1221]
## Description

I just moved to using Docker yesterday as I feel like it runs a little better compared to native on Windows. I noticed that I no longer have access to screenshots though. 

I assume I’d need to mount a volume to Docker to be able to store the screenshots on but honestly I am really new to Docker so I am not sure how. I’d appreciate it if someone tells me, and others how to make this happen (without having to rely on the container emailing the screenshots).

**Comment by rocketeer55** [comment]
You're on the right track with volumes. First you'd need to find the folder inside the container where the screenshots are being saved to. You can run bash inside the docker container (kinda like ssh'ing into it) with `docker exec -it <container_name> /bin/bash`, then you can move around inside there until you find the folder where the screenshots go. Remember that absolute path.

Then, in your docker run command, add another argument: `-v <absolute_path_on_pc>:<absolute_path_in_container>`. See [the documentation](https://docs.docker.com/storage/volumes/) for more details. But basically it'll link that local path with the given path inside the container. It will also make anything in that folder persist, even if you stop and remove the container.

**Comment by MRizkBV** [comment]
@rocketeer55 That worked! Thanks!

Since streetmerchant saves screenshots in the base folder that means I had to mount a volume for /app. Doing so in an empty folder makes it fail to start as it starts looking for the street merchant files in the mounted volume instead of the image. 

So to get around this issue I had to download a copy of the code here and put it in the folder mounted to Docker. Now it works fine, and screenshots gets put there. Yay!
